src/main/webapp/script.js

```diff
@@ -33,9 +35,47 @@ function init() {
 /**
  * Compiles the handlebars template representing a task.
  */
-function compileTaskTemplate() {
+function compileTemplates() {
   const taskTemplateElement = document.getElementById("task-template").innerHTML;
+  const selectOptionsTemplateElement = document.getElementById("select-options-template").innerHTML;
   taskTemplate = Handlebars.compile(taskTemplateElement);
+  selectOptionsTemplate = Handlebars.compile(selectOptionsTemplateElement);
+}
+
+const DEFAULT_WORKING_HOURS = {
+  "start-hour": "09",
```

Comment:
you can omit all the quotes around the keys. They're not necessary in JavaScript objects.

Reply:
The quotes are added because there are hyphens in the ids. I could use camelCase ids, but I would leave it as it is now.

Reply:
ah yes that is the issue then. Following the style guide you should use camelCase for object keys.

https://google.github.io/styleguide/jsguide.html#features-object-literals

Reply:
On the other hand you're relying on the logic to reuse the id of the form field in HTML which should use hyphens (as you do) according to the style guide: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters